src/coreclr/scripts/genEventPipe.py

```diff
@@ -618,6 +621,18 @@ def getMonoEventPipeHelperFileImplPrefix():
     size_t *size,
     bool *fixed_buffer);
 
+bool
```

Comment:
Will these new methods only be called when BIGENDIAN is defined?

Reply:
Yes, actually those are only called for big-endian archs, though, I implemented them endian independent just in case.